gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -165,6 +165,10 @@
         <module name="IllegalType"> <!-- Java Coding Guide: Limit coupling on concrete classes -->
             <property name="illegalClassNames" value="java.util.ArrayList, java.util.HashSet, java.util.HashMap, java.util.LinkedList, java.util.LinkedHashMap, java.util.LinkedHashSet, java.util.TreeSet, java.util.TreeMap, com.google.common.collect.ArrayListMultimap, com.google.common.collect.ForwardingListMultimap, com.google.common.collect.ForwardingMultimap, com.google.common.collect.ForwardingSetMultimap, com.google.common.collect.ForwardingSortedSetMultimap, com.google.common.collect.HashMultimap, com.google.common.collect.LinkedHashMultimap, com.google.common.collect.LinkedListMultimap, com.google.common.collect.TreeMultimap"/>
         </module>
+        <module name="IllegalType">
```

Comment:
Can you give this an ID so that people can suppress it if they encounter some weird case in which the guava one is absolutely necessary?

```diff
    <module name="IllegalType">
+       <property name="id" value="BanGuavaCaches"/>
```

Reply:
dan, you merged this before this change

Reply:
Ah whoops, I was just so excited about caffeine ;) any thoughts about the id? (I'll just push up another PR)

Reply:
id looks fine